### PR TITLE
refactor(Core/DB/SAI): move Gryphoneer Windbellow to SAI

### PR DESCRIPTION
**Pull request by vhiperdev**
## CHANGES PROPOSED:
-  Migrate the Gryphoneer Windbellow, NPC 20235, to SAI / database.

## ISSUES ADDRESSED:
- Closes 

## TESTS PERFORMED:
-  Behavior was the same as the current C ++ npc_taxi script
-

## HOW TO TEST THE CHANGES:

- ``.go cre 96987``
perform the missions described in the sql changes of this PR

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1592611528011042200.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+-- Migrate NPC to use SmartAI
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '', `gossip_menu_id` = 8096 WHERE `entry` = 20235;
+
+-- Condition: Gossip menu ID 0 needs either quest ID 10163 or quest ID 10346 to be incomplete.
+-- Condition: Gossip menu ID 1 needs quest ID 10382 to be complete
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`= 15 AND `SourceGroup` = 8096 AND `SourceEntry`IN (0,1);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`SourceId`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionTarget`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`,`NegativeCondition`,`ErrorTextId`,`ScriptName`,`Comment`) VALUES
+(15,8096,0,0,0,9,0,10163,0,0,0,0,'','Gryphoneer Windbellow - Show gossip option 0 if player has taken quest ID 10163'),
+(15,8096,0,0,1,9,0,10346,0,0,0,0,'','Gryphoneer Windbellow - Show gossip option 0 if player has taken quest ID 10346'),
+(15,8096,1,0,0,28,0,10382,0,0,0,0,'','Gryphoneer Windbellow - Show gossip option 1 if quest ID 10382 is completed');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1592611528011042200.sql:20]:
```suggestion
(15,8096,0,0,0,9,0,10163,0,0,0,0,'','Gryphoneer Windbellow - Show gossip option 0 if player has taken quest ID 10163 OR'),
(15,8096,0,0,1,9,0,10346,0,0,0,0,'','Gryphoneer Windbellow - Show gossip option 0 if player has taken quest ID 10346 AND'),
(15,8096,1,0,0,28,0,10382,0,0,0,0,'','Gryphoneer Windbellow - Show gossip option 1 if quest ID 10382 is completed');
```

**Comment by Kitzunu** [comment]
Nice work!

**Comment by FrancescoBorzi** [comment]
thanks @vhiperdev ! waiting for this to be tested

**Comment by diluh** [comment]
Tested work!

File: data/sql/updates/pending_db_world/rev_1592611528011042200.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+-- Create SmartAI for Gryphoneer Windbellow
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 20235);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(20235,0,0,2,62,0,100,0,8096,0,0,0,11,33899,0,0,0,0,0,7,0,0,0,0,0,0,0,'Gryphoneer Windbellow - On Gossip Option 0 Selected - Cast Spell 33899'),
+(20235,0,1,2,62,0,100,0,8096,1,0,0,11,35065,0,0,0,0,0,7,0,0,0,0,0,0,0,'Gryphoneer Windbellow - On Gossip Option 1 Selected - Cast Spell 35065'),
+(20235,0,2,0,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,'Gryphoneer Windbellow - On Linked Actions - Close Gossip');
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1592611528011042200.sql:27]: @vhiperdev just curious: are you using Keira3 to generate the SAI and the comments?

Comment (vhiperdev) [reply data/sql/updates/pending_db_world/rev_1592611528011042200.sql:27]: @FrancescoBorzi I didn't use

**Pull request merged**